UploadFile/Page/Pag50410.BCPUpload.al

```diff
@@ -0,0 +1,32 @@
+page 50410 "BCP Upload"
+{
+    PageType = ListPart;
+    ApplicationArea = All;
+    UsageCategory = None;
+    SourceTable = "BCP Upload";
+
+    layout
+    {
+        area(Content)
+        {
+            repeater("File Upload")
+            {
+
+                field("BCP File Name"; Rec."BCP File Name")
+                {
+                    ApplicationArea = All;
+                    ToolTip = 'Filename';
+                    trigger OnDrillDown()
+                    var
+                        downloadFunc: Codeunit "BCP UploadFile";
+                    begin
+                        CurrPage.Update();
+                        downloadFunc.DownloadFile(rec."BCP Table ID", rec."BCP No.", rec."BCP ID");
```

Comment:
So I think it's good to rename UploadFile codeunit to something like "Content Mgt."

Reply:
i think that is perfect